pages/common/sleep.md

```diff
@@ -14,3 +14,11 @@
 - Delay in hours:
 
 `sleep {{hours}}h`
+
+- Pause for 1 day 3 hours:
+
+`sleep 1d 3h`
```

Comment:
```suggestion
`sleep {{time}}`
```

Reply:
The idea is to show multiple parameters with different units are possible. And multiple parameters are added.